META-INF/keycloak-themes.json

```diff
@@ -3,6 +3,10 @@
     {
       "name": "keywind",
       "types": ["login"]
+    },
+    {
+      "name": "keywind",
+      "types": ["email"]
```

Comment:
It seems like changes like this should be part of another PR. 🙂

Reply:
done.